src/index.html

```diff
@@ -12,7 +12,40 @@
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__img"></div>
+      <div class="card__container">
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
```

Comment:
```suggestion
        </h2>
        
        <p class="card__code">Product code: 195434</p>
```

The same in all other places

<img width="1119" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/86789161/39381cd4-aa5d-4bca-9188-422e6883ae0c">